ialign.el

```diff
@@ -56,7 +57,10 @@
     (define-key map (kbd "C-c ?") #'ialign-show-help)
     map)
   "Keymap used in minibuffer during `ialign'."
-  :group 'ialign)
+  :group 'ialign
+  :type '(restricted-sexp :match-alternatives (keymapp)))
```

Comment:
From https://emacs.stackexchange.com/questions/62565/is-there-a-defcustom-type-for-a-keymap :slightly_smiling_face: